src/lib/wallet.js

```diff
@@ -16,26 +15,21 @@ import Logger from './logger';
 
 async function initMiddleware() {
   const { middlewareUrl } = store.getters.activeNetwork;
-  const swag = await fetchJson(`${middlewareUrl}/middleware/api`);
-  swag.paths['/names/auctions/{name}/info'] = {
-    get: {
-      operationId: 'getAuctionInfoByName',
-      parameters: [
-        {
-          in: 'path',
-          name: 'name',
-          required: true,
-          type: 'string',
-        },
-      ],
-    },
+
+  const fetchMiddleware = async (url, field) => (await fetchJson(url))[field];
```

Comment:
let's add middleware url if it has `middleware` in the name:
```suggestion
  const fetchMiddleware = async (path, field) => (await fetchJson(`${middlewareUrl}/${path}`))[field];
```